### ISSUE TITLE
tkListViewHeader and ListViewGridLayout with spanCount=2

### ISSUE THREAD

**Issue by miroslavaivanova** [telerik/nativescript-ui-feedback#94]
from @jet-lee-m

Hi,

I have a problem with a RadListView that has a ListViewGridLayout with a spanCount=2. When I add a tkListViewHeader, the header is displayed wrong on iOS-Devices. The width of the header is only on the half screen. On Android-Devices the header is displayed right on the full width of the screen.

Here is my layout code:
```html
<RadListView [items]='itemData'>
<template tkListItemTemplate let-item='item'>
...
</template>
<template tkListViewHeader >...</template>
<ListViewGridLayout tkListViewLayout scrollDirection='Vertical' spanCount="2"></ListViewGridLayout>
</RadListView>
```
I attached a screenshot of the iOS and Android screens. The Android screenshot is the desired behavior.
<img width="787" alt="image3" src="https://cloud.githubusercontent.com/assets/4198315/23709165/3a445c1c-0421-11e7-807d-adf5d5088d54.png">

Have you any idea?

Thanks

https://github.com/telerik/nativescript-ui-samples-angular/issues/53

**Comment by yezarela** [comment]
I'm having the same issue, any update for this?

**Comment by ginev** [comment]
@yezarela this seems to be a bug with the Grid layout. We will log it in our issue tracking system for fixing.

**Comment by yezarela** [comment]
Thanks @ginev, Good to hear that!

**Comment by dgomezs** [comment]
@ginev Is this fixed in the 1.7 release? Just trying to understand if it is worth waiting for the fix or shall I start changing my layout. Is there a work around it?

Thank you in advanced.
